### PR TITLE
Add Layers Icon

### PR DESCRIPTION
**Pull request by charlie-tf**
image

### CODE REVIEW

File: src/components/Icon/icons/index.tsx [hunk 1]
```diff
@@ -135,6 +136,7 @@ export type IconName =
   | 'iconTrashCan'
   | 'info'
   | 'key'
+  | 'layers'
```
Comment (karthikbtf) [inline src/components/Icon/icons/index.tsx:139]: rename to `iconLayers`? @chudo76 wants the icon names to match svg file names

File: src/components/Icon/icons/index.tsx [hunk 2]
```diff
@@ -222,6 +224,7 @@ const icons: Record<IconName, SVGComponent> = {
   iconTrashCan: IconTrashCan,
   info: Info,
   key: Key,
+  layers: Layers,
```
Comment (karthikbtf) [inline src/components/Icon/icons/index.tsx:227]: IconLayers

**Review by karthikbtf — APPROVED** [review]
lgtm! sorry missed this earlier..
some comments on naming

File: src/components/Icon/icons/index.tsx (hunk 1 above)
Comment (charlie-tf) [reply src/components/Icon/icons/index.tsx:139]: Sure thing!

**Pull request merged**